source/includes/documentation/_vue_js_elements.md

```diff
@@ -1413,67 +1413,107 @@ For examples of usage see: [JDI Vuetify Select tests](https://github.com/jdi-tes
+__Vuetify v2.6.14__ code example:
+```html
+<div class="v-input theme--light v-text-field v-text-field--is-booted v-file-input" id="MultipleFileInput">
+  <div class="v-input__prepend-outer">
+    <div class="v-input__icon v-input__icon--prepend">
+      <button type="button" aria-label="prepend icon" class="v-icon notranslate v-icon--link mdi mdi-paperclip theme--light"></button>
+    </div>
+  </div>
+  <div class="v-input__control">
+    <div class="v-input__slot">
+      <div class="v-text-field__slot">
+        <label for="input-145" class="v-label theme--light" style="left: 0px; right: auto; position: absolute;">File input</label>
+        <div class="v-file-input__text"></div>
+        <input id="input-145" type="file" multiple="multiple">
+      </div>
+      <div class="v-input__append-inner"><div>
+      </div>
+      </div>
+    </div>
+    <div class="v-text-field__details">
+      <div class="v-messages theme--light">
+        <div class="v-messages__wrapper"></div>
+      </div>
+    </div>
+  </div>
+</div>
+```
 
+|Method | Description                                | Return Type
+--- |--------------------------------------------| ---
+**isMultiply()** | Shows that element can take multiply files | boolean
+**textInputField()** | Returns text input area                    | UIElement
+**counter()** | Returns counter                            | UIElement
+**prefix()** | Returns prefix                             | UIElement
+**suffix()** | Returns suffix                             | UIElement
+**prependOuterIcons()** | Returns list of prepend outer icons        | List\<Icon>
+**prependInnerIcons()** | Returns list of prepend inner icons        | List\<Icon>
+**appendInnerIcons()** | Returns list of append inner icons         | List\<Icon>
+**appendOuterIcons()** | Returns list of append outer icons         | List\<Icon>
+**getPrependOuterIcon()** | Returns first prepend outer icon           | Icon
+**getPrependInnerIcon()** | Returns first prepend inner icon           | Icon
+**getAppendInnerIcon()** | Returns first append inner icon            | Icon
+**getAppendOuterIcon()** | Returns first append outer icon            | Icon
+**getText()** | Returns text                               | String
+**label()** | Returns label                              | Label
+**placeholder()** | Returns placeholder text                   | String
+**setText(String text)** | Set text                                   | void
+**sendKeys(CharSequence... value)** | Add text                                   | void
+**clear()** | Clear text area                            | void
+**focus()** | Set mouse to text area                     | void
+**isDisplayed()** | Check if element is displayed              | boolean
+**isDisabled()** | Check if element is disabled               | boolean
```

Comment:
why 'isDisabled' repeated 2 times? :)

Reply:
not 2 time
1- isDisplayed
2- isDisabled

Reply:
oh) thnx